lib/shared/addon/components/managed-import-cluster-info/component.js

```diff
@@ -19,8 +19,25 @@ export default Component.extend({
   editing:     false,
   configField: 'k3sConfig',
 
+  initialKubernetesVersion: null,
   supportedK8sVersionRange: alias(`settings.${ C.SETTING.VERSION_K8S_SUPPORTED_RANGE }`),
 
+  init() {
+    this._super(...arguments);
+
+    this.set('initialKubernetesVersion', get(this, `config.kubernetesVersion`))
+  },
+
+  showKubernetesVersionBanner: computed('config.kubernetesVersion', 'initialKubernetesVersion', function() {
+    const initialVersion = get(this, 'initialKubernetesVersion');
+    const selectedVersion = get(this, 'config.kubernetesVersion');
+
+    const initialIsGte127 = initialVersion && Semver.gte(initialVersion, '1.27.0');
```

Comment:
Wouldn't we want this to be lt(..., '1.27.0') since the banner should show be shown to anyone upgrading to 1.27 or above?

Reply:
Yeah, that's more straightforward than than using gte and then testing for the inverse below.. should be fixed now